extras/createdevproject.py

```diff
@@ -0,0 +1,47 @@
+"""
+Modifies the project template to fit with local development
+"""
+import os
+import sys
+from misago.core import setup
+
+
+PROJECT_NAME = 'forum'
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+def main():
+    global PROJECT_NAME, SETTINGS_FILE
+
+    # Allow for overriding project name
+    if len(sys.argv) > 1:
+        PROJECT_NAME = sys.argv[1]
+    else:
+        sys.argv.append(PROJECT_NAME)
+
+    settings_file = os.path.join(BASE_DIR, PROJECT_NAME, PROJECT_NAME, 'settings.py')
+
+    # Avoid recreating if already present
+    if os.path.exists(settings_file):
+        return
+
+    setup.start_misago_project()
+    tweak_settings(settings_file)
+
+
+def tweak_settings(f):
```

Comment:
Lets rename this to `fill_in_settings`. `tweak_settings` doesn't seem to be truthful to function's contents.